configure

```diff
@@ -2182,7 +2177,7 @@ else
   inf "Source-level replay debugger: not supported"
 fi
 
-if test "$debugruntime" = "runtimed"; then
+if test $debugruntime; then
```

Comment:
The call to `test` is redundant here because `true` and `false` are shell commands.